src/SidebarSectionItem/docs/examples.js

```diff
@@ -23,12 +23,20 @@ export const prefix = `
 <SidebarSectionItem prefix={<Box width="8px" height="8px" borderRadius="50%" backgroundColor="G10"></Box>}>Action</SidebarSectionItem>
 `;
 
-export const suffix = `
+export const counterBadgeSuffix = `
 <SidebarSectionItem suffix={<CounterBadge skin="standard">5</CounterBadge>}>Action</SidebarSectionItem>
 `;
 
+export const badgeSuffix = `
+<SidebarSectionItem suffix={<Badge skin="standard" size="small">New</Badge>}>Action</SidebarSectionItem>
+`;
+
 export const lightSkin = `
-<Sidebar skin="light">
-  <SidebarSectionItem>Action</SidebarSectionItem>
-</Sidebar>
+<Box height="36px" overflow="hidden">
```

Comment:
After we remove the `min-height: 480px;` the `overflow="hidden"` becomes redundant also:
```suggestion
<Box height="36px">
```

Reply:
Depends on the resolution on the sidebar minimum height.